src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -378,10 +378,11 @@ private void resetCache()
 
 		final IMEInventory<IAEItemStack> out = this.getInternalHandler();
 
-		if( this.monitor != null )
-		{
-			this.monitor.onTick();
-		}
+		// TODO: evaluate, if this is really (not) necessary.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.orod.de/coding_rules#rule_key=squid%3AS1135)